onformsubmit-http/index.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * Handles Google Form submission answers
+ */
+const axios = require("axios").default;
+
+const Auth = require("./utils/auth");
+const Submission = require("./utils/submission");
+const Firecloud = require("./utils/firecloud");
+
+// default consts
+// make changes as needed
+const SERVICE_ACCOUNT_KEY = "creds.json";
+const FIRECLOUD_URL = "https://api.firecloud.org";
+// const FIRECLOUD_URL =
+//   "https://firecloud-orchestration.dsde-dev.broadinstitute.org/";
+const ADMIN_EMAILS = [
+  "vreeves@broadinstitute.org",
+  "candace@broadinstitute.org",
+  "ltran@asymmetrik.com",
+];
+
+exports.onFormSubmit = async (req, res) => {
+  const message = req.body;
+
+  // initialize axios with token
+  let auth, serviceAccountEmail;
+  try {
+    auth = new Auth(SERVICE_ACCOUNT_KEY);
+    serviceAccountEmail = auth.clientEmail();
+
+    const bearerToken = await auth.requestAccessToken();
+    axios.defaults.baseURL = FIRECLOUD_URL;
```

Comment:
you should write a custom instances instead of editing the global axios obj https://axios-http.com/docs/config_defaults. it will be better for readability instead referencing axois